a3p-integration/proposals/z:acceptance/governance.test.js

```diff
@@ -17,8 +17,17 @@ const governanceDriver = await makeGovernanceDriver(fetch, networkConfig);
 test.serial(
   'economic committee can make governance proposal and vote on it',
   async t => {
+    const vaultFactoryParamsBefore = await readLatestHead(
+      'published.vaultFactory.governance',
+    );
+    const newChargingPeriod =
+      // @ts-expect-error it's a record
+      vaultFactoryParamsBefore.current.ChargingPeriod.value === 400n
```

Comment:
Rather than a `ts-expect-error` here at the point of usage, I suggest fixing the type at the point of declaration. A `ts-expect-error` there may or may not be necessary.